baseplate/_compat.py

```diff
@@ -23,6 +23,9 @@
     def iteritems(d):
         return iter(d.items())
 
+    def reraise_exception(exc, exc_info):
+        raise exc
```

Comment:
oh wow, TIL this is all better in py3. that's fantastic. https://legacy.python.org/dev/peps/pep-3134/

given the impending doom of py27, do we want to just drop this compat method and let it be a shitty traceback on 2.7?

Reply:
yeah, I think that'd be better than this goofy workaround